MaxMind.MinFraud.UnitTest/project.json

```diff
@@ -23,25 +12,19 @@
     },
     "dotnet-test-nunit": "3.4.0-beta-1",
     "MockHttpSigned": "1.3.0-netstandard-alpha3",
-    "NETStandard.Library": "1.6.0",
     "NUnit": "3.2.1"
   },
   "testRunner": "nunit",
   "frameworks": {
-    "net451": {},
-    "netstandard1.6": {
-      "imports": [
-        "dnxcore50",
-        "netcoreapp1.0",
-        "portable-net45+win8"
-      ],
+    "net451": { },
+    "netcoreapp1.0": {
+      "imports": "portable-net45+win8",
       "dependencies": {
         "Microsoft.NETCore.App": {
           "version": "1.0.0-*",
           "type": "platform"
-        },
-        "System.Console": "4.0.0"
+        }
       }
     }
   }
-}
+}
```

Comment:
Yeah, I edited it in VS. Although I couldn't find an option for force EOF newlines in VS proper, CodeMaid did have an option so hopefully we won't keep switching back and forth going forward.